releasenotes/notes/tranform-API-da38b196c1c72d44.yaml

```diff
@@ -0,0 +1,6 @@
+---
+features:
+  - |
+    It's now possible to made some transformation to the stored timeseries
+    during the retrieval of the measures. The allowed transformations are
+    absolute and negative.
```

Comment:
it's not clear but is negative like the opposite of absolute? ie. -5 is still -5 if negative transform?